0ME9A/password
# Add Password History and Settings Features

**Title:** Add Password History and Settings Features

Hello,

I've implemented the "Password History" and "Settings" features as discussed in the feature request. Here are the details of the changes:

**Changes Made:**

1. Added a new state variable to store the history of generated passwords.
2. Created a new component to display the password history.
3. Integrated the password history component into the main password generator component.
4. Added functionality to update the password history state whenever a new password is generated.
5. Added a delete history functionality that allows users to clear their password history.
6. Created a new "Settings" component that includes theme settings and a toggle for the password salt option.
7. Moved the password salt option from the main password generator component to the "Settings" component.
8. Added a "History" button to the header, which is displayed by default.

**How to Test:**

1. Generate a new password and check if it appears in the password history section.
2. Use the delete history functionality and ensure that the password history is cleared.
3. Navigate to the "Settings" component and test the theme settings and password salt option.
4. Ensure that the "History" button is displayed in the header by default.

I believe these features enhance the user experience by providing more customization options and a way to access and manage previously generated passwords. I've thoroughly tested these changes and everything seems to be working well. However, I'd appreciate if you could review my code and provide any feedback.

Thank you for considering this pull request. I'm looking forward to your feedback.

Best,
@0ME9A
